custom/elements.json

```diff
@@ -568,29 +568,8 @@
     "global_attributes": {
       "attributes": [
         {"class": "className", "tabindex": "tabIndex"},
-        "clip",
-        "color",
-        "cursor",
-        "data",
-        "direction",
-        "display",
-        "fill",
-        "filter",
-        "href",
         "id",
-        "kerning",
-        "lang",
```

Comment:
`lang` is listed in https://svgwg.org/svg2-draft/struct.html#CommonAttributes but I think it's actually not possible to test because there is no reflected IDL attributes.